contracts/TokenSale.sol

```diff
@@ -40,7 +43,7 @@ contract TokenSale is FinalizableCrowdsale, Pausable {
      * @param _companyToken ERC20 contract address that has minting capabilities
      * @param _rate The token rate per ETH
      * @param _starRate The token rate per STAR
-     * @param _wallet Multisig wallet that will hold the crowdsale funds.
+     * @param _wallet FundsSplitter wallet that redirects funds to client and Starbase.
```

Comment:
Comment is a bit misleading as it is not enforcing FundsSplitter. Previous comment had also the same issue as there was not a a Multisig check. I would write that this is an address where funds go to

Reply:
Actually I see that that it must be FundsSplitter.  You added the interface 👍